ocaml/parsing/parser.mly

```diff
@@ -150,6 +150,8 @@ let mk_attr ~loc name payload =
   Builtin_attributes.(register_attr Parser name);
   Attr.mk ~loc name payload
 
+(* CR zqian: someday we should put attributes on the expression, instead of
+extension node. *)
```

Comment:
Why?

Reply:
Since we put modes in `ppat_attributes` to means "this pattern should be of this mode", and expressions are dual to patterns, we should put modes in `pexp_attributes` to mean "this expression should be of this modes".

But I also understand this is a big change and would need some discussion.

Reply:
I think I'm just arguing to delete this CR as the point is not that important. If it ever becomes important to make the attribute encoding the same, we can make that change then.